.vscode/settings.json

```diff
@@ -20,5 +23,7 @@
       "url": "./.vscode/package.json-schema.json"
     }
   ],
+  "tailwindCSS.classFunctions": ["classnames", "classNames"],
+  "tailwindCSS.validate": true,
```

Comment:
Enables Tailwind's built-in linting to validate CSS classes and catch potential errors